### PR TITLE
chore: add translations for 1.8.0

### PR DESCRIPTION
**Pull request by scottdover**
**Summary**
This updates the translations needed for our 1.8.0 release.  There are two ways to update translations.
 - You can add suggestions for each translated string. [click here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) for more information on adding suggestions.
 - You can update translations using one of the csvs below by editing the file and re-attaching it to this pull request
    - **de**: [de-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602958/de-locale-update-1.8.0.csv)
    - **es**: [es-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602959/es-locale-update-1.8.0.csv)
    - **fr**: [fr-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602960/fr-locale-update-1.8.0.csv)
    - **it**: [it-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602962/it-locale-update-1.8.0.csv)
    - **ja**: [ja-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602963/ja-locale-update-1.8.0.csv)
    - **ko**: [ko-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602964/ko-locale-update-1.8.0.csv)
    - **pt-br**: [pt-br-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602966/pt-br-locale-update-1.8.0.csv)
    - **zh-cn**: [zh-cn-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14602967/zh-cn-locale-update-1.8.0.csv)

**Testing**
 - [x] Verify all strings are captured by running the translation check (see https://github.com/sassoftware/vscode-sas-extension/actions/runs/8327405145)
 - [x] Verify **de** strings are translated
 - [x] Verify **es** strings are translated
 - [x] Verify **fr** strings are translated
 - [x] Verify **ja** strings are translated
 - [x] Verify **ko** strings are translated
 - [x] Verify **zh-cn** strings are translated
 - [x] Verify **it** strings are translated
 - [x] Verify **pt-br** strings are translated

### CODE REVIEW

**Comment by SASJedi** [comment]
Portuguese updates attached
[pt-br-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14603244/pt-br-locale-update-1.8.0.csv)

**Comment by Criptic** [comment]
[de-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14603258/de-locale-update-1.8.0.csv)

**Comment by meilanji** [comment]
[ko-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14611294/ko-locale-update-1.8.0.csv)

Added the Korean translations.

**Comment by dfxvir** [comment]
[fr-locale-update-1.8.0.csv](https://github.com/sassoftware/vscode-sas-extension/files/14618356/fr-locale-update-1.8.0.csv)

File: l10n/bundle.l10n.zh-cn.json [hunk 1]
```diff
@@ -108,12 +108,14 @@
   "The output file name must end with the .flw extension.": "输出文件名必须以 .flw 扩展名结尾。",
   "The {selected} SAS connection profile has been deleted from the settings.json file.": "已从settings.json文件中删除{selected} SAS连接配置文件。",
   "There was an error executing the SAS Program. See [console log](command:workbench.action.toggleDevTools) for more details.": "SAS程序执行出错。请参见[开发控制台日志](command:workbench.action.toggleDevTools)获取更多信息。",
+  "This platform does not support this connection type.": "This platform does not support this connection type.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:111]:
```suggestion
  "This platform does not support this connection type.": "该平台不支持该连接类型。",
```

File: l10n/bundle.l10n.zh-cn.json [hunk 2]
```diff
@@ -108,12 +108,14 @@
...
   "Unable to add file to my folder.": "无法将文件添加到我的文件夹。",
   "Unable to create file \"{name}\".": "无法创建文件\"{name}\"。",
   "Unable to create folder \"{name}\".": "无法创建文件夹\"{name}\"。",
   "Unable to delete file.": "无法删除文件。",
   "Unable to delete folder.": "无法删除文件夹。",
   "Unable to delete table {tableName}.": "无法删除表 {tableName}。",
+  "Unable to download files.": "Unable to download files.",
```
Comment (scnwwu) [inline l10n/bundle.l10n.zh-cn.json:118]:
```suggestion
  "Unable to download files.": "无法下载文件。",
```

**Review by smorrisj — APPROVED** [review]
LGTM. Primarily just made sure we didnt have any default placeholders still in place.

**Pull request merged**